src/java/transform/visitors/TransformVisitor.java

```diff
@@ -575,75 +578,109 @@ public boolean visit(MethodInvocation node) {
 	// expr rule
```

Comment:
Using this to create a conversation to be resolved and point out a location that a comment would be great

Reply:
@charles-moloney you asked about comments for overridden methods or just our own. With some of the code that exists the original method has a great generic description so it may be enough on its own though, I am personally fond of having updated comments for all modified or updated code to make the use case for it clear. 

When testing comments and on hovers I got mixed results depending on IDE. /** */ always seemed to work and often allowed partial overwrites so less reiterations. /// usually worked but concatenated the lines. // sometimes worked but not always. I cannot test every IDE but that was my findings. May be worth more research but I don't want to download more IDE's at the moment to test and was finding limited knowledge online